forward_command_controller/src/forward_command_controller.cpp

```diff
@@ -82,7 +82,7 @@ CallbackReturn ForwardCommandController::on_configure(
       rt_command_ptr_.writeFromNonRT(msg);
     });
 
-  RCLCPP_INFO_STREAM(get_node()->get_logger(), "configure successful");
+  RCLCPP_INFO(get_node()->get_logger(), "configure successful");
```

Comment:
When you are already editing this, can you take a look at the following use?

```suggestion
  RCLCPP_INFO(node_->get_logger(), "configure successful");
```